aip/0233.md

```diff
@@ -0,0 +1,82 @@
+---
+aip:
+  id: 233
+  state: draft
+  created: 2019-06-18
+permalink: /233
+redirect_from:
+  - /0233
+---
+
+# Batch Create
+
+Some APIs need to allow users to create multiple resources in a single
+transaction. A batch create method provides this functionality.
+
+## Guidance
+
+Batch create methods are specified using the following pattern:
```

Comment:
```suggestion
APIs **may** support Batch Create using the following pattern:
```

Reply:
I will incorporate this in my own commit